src/components/molecules/Stepper/Stepper.tsx

```diff
@@ -18,21 +19,15 @@ export type StepperProps = {
   border?: boolean
 }
 
-const defaultStepperProps = {
-  title: null,
-  elements: [],
-  minHeight: 0,
-}
-
-export const Stepper = ({
-  elements,
-  title,
-  minHeight,
+export default function Stepper({
```

Comment:
why using export default here?

Reply:
I prefer to do it that way, but I was testing some stuff... I has some changes on this PR that need to be reverted